### PR TITLE
Added app_environment and business_unit tag to subnet role

### PR DESCRIPTION
**Pull request by zoiec**
🐘  x 2

- Added app_environment and business_unit tag to role

### CODE REVIEW

File: tasks/main.yml [hunk 1]
```diff
@@ -5,13 +5,15 @@
...
     resource_tags:
-      Name: '{{ item.Name }}'
-      Application: '{{ item.app_name }}'
-      Environment: '{{ item.app_environment }}'
-      ManagedByAnsible: '{{ ansible_vpc_subnets_tags_ManagedByAnsible }}'
-  with_items: '{{ subnets }}'
+      Name: "{{ item.Name }}"
+      Application: "{{ item.app_name }}"
+      Environment: "{{ item.app_environment }}"
+      app_environment: "{{ item.app_environment }}"
```
Comment (MathieuGilbert) [inline tasks/main.yml]: Did you intend to add the `app_environment` tag, or did it just get missed from the commit/pr messages?

**Comment by MathieuGilbert** [comment]
💩

File: tasks/main.yml (hunk 1 above)
Comment (zoiec) [inline tasks/main.yml]: I intended to add it also, I didn't want to remove the Environment tag just in case someone is using it for something

**Comment by zoiec** [comment]
👋

**Comment by MathieuGilbert** [comment]
:elephant: :elephant: :elephant: 1/2

**Comment by jessedoyle** [comment]
:elephant::elephant::elephant: (2/2)

**Review by jessedoyle — APPROVED** [review]

**Pull request merged**